newrelic_lambda_cli/api.py

```diff
@@ -401,6 +402,16 @@ def enable_lambda_integration(gql, input, linked_account_id):
             "[%d] in New Relic account [%d]." % (linked_account_id, input.nr_account_id)
         )
         return False
+
+    metric_collection_mode = account["metricCollectionMode"]
+
+    if metric_collection_mode == "PUSH":
```

Comment:
This appears to bypass enabling Lambda for the linked cloud account. Does metrics streaming include Lambda?